### ISSUE TITLE
Inconsistency issues

### ISSUE THREAD

**Issue by HelveticaStandardBold** [jonathangarelick/tick-fixer#8]
I am currently having inconsistency issues with tick fixer and it's ability to improve my tick health. Some of the time it still sits at about 80%. I've confirmed that it's not on my end. Anyone else been having this issue? I'm wondering if it's not just the jagex servers at this point. Thanks in advance.

**Comment by jonathangarelick** [comment]
Hi, thanks for filing an issue. This is the correct place for problems with the plugin.

When you say "confirmed it's not on my end" do you mean that it works when you manually ping?

**Comment by HelveticaStandardBold** [comment]
Yes it works when i manually ping/have a livestream running i the background, sorry for not specifying. Sometimes i'll hop worlds and it'll all of a sudden be working. I'm wondering if it's not just a bug with the tick health tracker not updating properly.

**Comment by jonathangarelick** [comment]
Okay. I have a pending update that will let you configure both the target IP address and the ping interval, which might solve your issue. In the meantime, I will spend some time looking into this.

**Comment by HelveticaStandardBold** [comment]
Thank you. I'm sorry i can't give more info, its very inconsistent and for all i know it could just be a visual bug with the updating of the health plugin.

**Comment by jonathangarelick** [comment]
Hi @HelveticaStandardBold , I just released a new version of the plugin that allows you to configure the target IP address and interval.

Here are two suggestions that might improve the consistency:
- Try reducing the ping interval to around 150ms (the minimum is 100ms)
- Try using another IP address, like Google DNS (8.8.8.8)

Hope this helps.
